server/form-pages/apply/schema.json

```diff
@@ -1660,16 +1660,36 @@
           }
         },
         "plans-in-place": {
-          "type": "object",
```

Comment:
Good catch on this! We should revisit what we're actually doing with these schemas, because I think we're thinking about handling this in a less strict way (i.e. not using JSON schema), but nothing's been codified yet.

Reply:
Just running `script/test` did this for me. 
Yeah I think it would be good to get rid of the JSON schema dependency